src/app/users/users-update/users-update.component.html

```diff
@@ -3,7 +3,10 @@
 </mat-toolbar>
 
 <div class="space-container">
-  <mat-toolbar class="primary-color font-size-1" i18n>Edit Your Profile</mat-toolbar>
+  <mat-toolbar class="primary-color font-size-1">
+    <span>{{user.name}}</span><span class="toolbar-fill"></span>
+    <span class="margin-lr-3">Edit Profile</span>
```

Comment:
this code is not required and also will not look nice as it is just text which shows in the place used for button